temp_workspace/input/path_config.yaml

```diff
@@ -0,0 +1,5 @@
+  pdb_dir: '../temp_workspace/input/'
+  micrograph_keyword: '_randomrot'
+  pdb_keyword: '4v6x'
```

Comment:
ok, so this would be an example .yaml file, and usually the user would have different hard coded values instead of `_randomrot` and `4v6x``

Reply:
Correct!